ProcessLib/ComponentTransport/ComponentTransportProcess.cpp

```diff
@@ -174,6 +174,32 @@ void ComponentTransportProcess::
         _local_assemblers, pv.getActiveElementIDs(), _coupled_solutions);
 }
 
+void ComponentTransportProcess::extrapolateIntPtProcessSolutions(
+    const double t, std::vector<GlobalVector*>& x,
+    std::vector<GlobalVector*> const int_pt_x)
+{
+    auto& extrapolator = getExtrapolator();
+    auto const extrapolatables = NumLib::makeExtrapolatable(
+        _local_assemblers,
+        &ComponentTransportLocalAssemblerInterface::getIntPtLocalSolution);
+
+    assert(_process_data.chemical_process_data);
```

Comment:
The variable is not used in the function. Why checking?

Reply:
removed.